jujugui/static/gui/src/app/templates/storefront-search-row.partial

```diff
@@ -0,0 +1,113 @@
+<div class="{{#if nameWidth}}{{nameWidth}}{{else}}four-col{{/if}} charm-name__column">
+  <h3 class="list-block__list--item-title">
+    {{#compare docType 'charm'}}
```

Comment:
Good catch, seems to be a bug in the ported template, which checked `docType` instead of the correct `type`. Consequently it never evaluates to true. I'll delete.